client/client.go

```diff
@@ -18,6 +18,8 @@ type Client interface {
 	BulkIndexClose(context.Context) error
 	Checker(ctx context.Context, state *health.CheckState) error
 	CreateIndex(ctx context.Context, indexName string, indexSettings []byte) error
+	DeleteDocument(ctx context.Context, index, documentID string) error
```

Comment:
Small point for consistency, this parameter is called `indexName` in other functions.
```suggestion
	DeleteDocument(ctx context.Context, indexName, documentID string) error
```